hca/upload/api_client.py

```diff
@@ -8,8 +10,11 @@ class ApiClient:
     def __init__(self, deployment_stage):
         self.api_url_base = ApiClient.UPLOAD_SERVICE_API_URL_TEMPLATE.format(deployment_stage=deployment_stage)
 
-    def list_area(self, area_uuid):
-        url = "{api_url_base}/area/{uuid}".format(api_url_base=self.api_url_base, uuid=area_uuid)
-        response = requests.get(url)
-        assert response.ok
-        return response.json()['files']
+    def files_info(self, area_uuid, file_list):
+        url = "{api_url_base}/area/{uuid}/files_info".format(api_url_base=self.api_url_base, uuid=area_uuid)
+        response = requests.put(url, data=(json.dumps(file_list)))
+        if not response.ok:
+            raise RuntimeError("GET {url} returned {status}, {content}".format(url=url,
+                                                                               status=response.status_code,
```

Comment:
recommend putting the first parameter on the next line, since that ensures you don't need to re-indent all the lines if you change the first line in the future.

Reply:
Good idea. Will do.